src/opencmw_worker/CMakeLists.txt

```
@@ -54,7 +54,9 @@ enable_sanitizers(project_options)
 # allow for static analysis options
 include(cmake/StaticAnalyzers.cmake)
 if (CMAKE_BUILD_TYPE STREQUAL "Debug")
```

```suggestion
if ((CMAKE_BUILD_TYPE STREQUAL "Debug") AND (CMAKE_CXX_COMPILER_ID MATCHES "GNU"))
```
Having two nested if's increases the number of branches to cover. In this case the outer else is not reached for Debug builds.